src/inspector/view/qml/MuseScore/Inspector/notation/sectionbreaks/SectionBreakSettings.qml

```diff
@@ -62,7 +63,8 @@ Column {
 
     PropertyCheckBox {
         id: startWithLongInstrNames
-        text: qsTrc("inspector", "Start new section with long instrument names")
+        //: Watch out for string length and inspector width
+        text: qsTrc("inspector", "Start new section with\nlong instrument names")
```

Comment:
Please don't add hard-coded newlines to UI strings.

1. The result doesn't look good.
    ![image](https://github.com/musescore/MuseScore/assets/11011881/69a741c7-fd9a-4e9b-a882-7a255d7dbeaf)
2. It's not even necessary. Qt will wrap the string automatically if it's too long. (See [`wrapMode: Text.Wrap`](https://doc.qt.io/qt-6/qml-qtquick-text.html#wrapMode-prop))
    ![image](https://github.com/musescore/MuseScore/assets/11011881/5610846f-9574-4a3b-ac1f-0d1b9c487930)
    I had to write 'looooonnng' because the normal string actually fits at the default text size, so there is no need to wrap it at all!

Also, I notice that you added a line break to this string in the English US and English GB translations. Please don't use the English translations to fix UI bugs (not that there is a bug in this case). If you think there's a UI bug, open an issue for it.

Please remove the line break here, and also remove it from the translations for this string and any others like it.

Reply:
Ah, that must be new with Mu4 then and a leftover for a fix needed in Mu3.